client/core/simnet_trade.go

```diff
@@ -1331,7 +1336,8 @@ func (s *simulationTest) checkAndWaitForRefunds(ctx context.Context, client *sim
 		if !client.isSeller {
 			swapAmt = calc.BaseToQuote(match.Rate, match.Quantity)
 		}
-		refundAmts[tracker.wallets.fromAsset.ID] += int64(swapAmt)
+		assetID := tracker.wallets.fromAsset.ID
+		refundAmts[assetID] += int64(swapAmt)
```

Comment:
unintended change

Reply:
So TODO? The change is fine to me unless you meant it shouldn't do this increment?

Reply:
I just mean about declaring the assetID variable when it is only used once.